packages/babel-parser/test/fixtures/typescript/class/accessor-with-type/input.ts

```diff
@@ -0,0 +1,3 @@
+class Foo {
+  accessor prop: number = 1;
```

Comment:
Can you add more test cases? So we can cover cases like static, private, computed, etc.